### PR TITLE
Jcm content guides

### PR DESCRIPTION
**Pull request by cmcfadzean**
## Description
Updating the content sections to include what we currently have as content guides and a new vocab list.

### CODE REVIEW

File: app/views/pages/vocabulary.html.erb [hunk 1]
```diff
@@ -1,6 +1,163 @@
...
+        <a href="#crm" class="quick-links__link">CRM</a>
+      </li>
+    </ul>
+  </div>
+<% end %>
+
+<%= md("
+## The Category
+
+##### Knowledge Commerce
```
Comment (philschanely) [inline app/views/pages/vocabulary.html.erb]: Could we do `###` here or `<h3 class="t-sage-heading-5">`?

File: app/views/pages/content.html.erb [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+<% end %>
+
+<%= md("
+### Speak To Their Needs
+
+Not everyone is a confident writer, but everyone can improve their content by making sure it responds to the needs of our customers. Keep in mind that real people rely on Kajabi every day to run their businesses. The product, feature, or app you’re building can make a big difference to the people using it. Take some time to learn about who they are, what they need, and the language they use.
+
+### Keep It Simple
+
+Kajabi users are busy people who may be running their online business in addition to many other things going on in their life. They’re also located all over the world, have varying levels of literacy, and some may not speak English as their first language. Some content can be translated poorly and can be misinterpreted.
```
Comment (philschanely) [inline app/views/pages/content.html.erb]: Consider `Kajabi customers` (Kenny said we don't call them users)` and then `...They’re also located all over the world and have varying levels of literacy. Some may not speak English as their first language.`

File: app/views/pages/vocabulary.html.erb [hunk 3]
```diff
@@ -1,6 +1,163 @@
...
+
+The practice of charging customers for access to our knowledge. This could be through an online course, an e-book, a membership site, or any other digital product designed to share knowledge. Any commercial enterprise that exchanges knowledge for cash.
```
Comment (philschanely) [inline app/views/pages/vocabulary.html.erb]: `Knowledge commerce is the practice...` and `...to share knowledge. It's any commercial enterprise...`

File: app/views/pages/vocabulary.html.erb [hunk 4]
```diff
@@ -1,6 +1,163 @@
...
+      </li>
+      <li class="quick-links__item">
+        <a href="#crm" class="quick-links__link">CRM</a>
+      </li>
+    </ul>
+  </div>
+<% end %>
+
+<%= md("
+## The Category
```
Comment (philschanely) [inline app/views/pages/vocabulary.html.erb:38]: OK I see this section seems to be a series of definitions. Each of these to me could use a little more framing or lead-in as suggested in the first one. Keeping the "content strategy" points in mind here.

File: app/views/pages/voice_tone.html.erb [hunk 5]
```diff
@@ -1,6 +1,74 @@
...
 # Voice & Tone
-<p class="docs-heading__lead">Coming Soon...</p>
+<p class="docs-heading__lead">One way we write impactful content is by being aware of our voice and our tone. The section explains the difference between voice and tone, and lays out the elements of each as they apply to Kajabi.</p>
 )) %>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<%= md("
+What’s the difference between voice and tone? Think of it this way; You have the same voice all the time, but your tone changes. You might use one tone when you’re out to dinner with your closest friends, and one tone with you are in a meeting with your boss.
```
Comment (philschanely) [inline app/views/pages/voice_tone.html.erb]: `...this way: you...`and `...one time when you are in a meeting...`

File: app/views/pages/voice_tone.html.erb [hunk 6]
```diff
@@ -1,6 +1,74 @@
...
+
+The same is true for Kajabi. Our voice doesn’t change much from day-to-day, but our tone changes all the time.
+
+## Our Voice
+
+Voice is how we write and speak all the time as the Kajabi brand.
+
+At Kajabi, we’ve been creating solutions for 10 years, and accordingly, we consider ourselves experts on online businesses.
+
+That’s why we speak like a highly experienced mentor.
```
Comment (philschanely) [inline app/views/pages/voice_tone.html.erb]: Combine with last paragraph?

File: app/views/pages/voice_tone.html.erb [hunk 7]
```diff
@@ -1,6 +1,74 @@
...
+
+Over the years, we’ve put a lot of thought into the problems that our customers are facing. We’ve created a system that not only solves those problems but provides such a profound sense of relief that success is possible.
+
+We aren’t just thinking about the problems they’re facing now, but the problems they’re going to face when they grow, and we have solutions in place for that too.
+
+One thing we don’t do is solve perceived problems. No spending time worrying about “what if’s?” that won’t ever happen. We’re in the business of solving actual problems. Our focus is on doing, not overplanning.
```
Comment (philschanely) [inline app/views/pages/voice_tone.html.erb:25]: Just saying... does this mean we're reactionary? In other words, do we only wait until there is a problem before we do something? Surely we also plan new things to help our users. Surely we think ahead a bit to avoid possible problems. Is there a way to reword this to avoid that perception?

File: app/views/pages/voice_tone.html.erb [hunk 8]
```diff
@@ -1,6 +1,74 @@
...
+
+That means we’ll always have our customers’ best outcomes in mind when creating feature sets, whether they themselves agree with it or not.
```
Comment (philschanely) [inline app/views/pages/voice_tone.html.erb:27]: 🤔 This feels weird to me. Are we god?

File: app/views/pages/voice_tone.html.erb [hunk 9]
```diff
@@ -1,6 +1,74 @@
...
+
+And everything at Kajabi begins and ends with our customers’ success.
+
+We know how difficult the journey as a business owner is, so we’re here as our customers’ business partners, to guide them and make the path to success as painless as possible.
+
+## Our Tone
+
+- Positive
```
Comment (philschanely) [inline app/views/pages/voice_tone.html.erb:35]: I have found it helpful to do these as `this yet that` pairs that shows balance. Sometimes this would mean combining so items: `Positive yet Realistic` and `Bold yet empathetic`. Other times it might mean a balancer that voids the need for comment such as `Rebellious yet purposeful`

**Review by philschanely — APPROVED** [review]
Suggestions below, generally non-blocking. :)

File: app/views/pages/voice_tone.html.erb (hunk 9 above)
Comment (cmcfadzean) [reply app/views/pages/voice_tone.html.erb:35]: I think all these are great points. These are actually the guidelines that marketing put together and I would like to have our new UX writer take a stab at cleaning these up.
